### ISSUE TITLE
_is_xfce() crashes with Python 3

### ISSUE THREAD

**Issue by akdor1154** [n1k0/SublimeHighlight#67]
on line 138 of HighlightLib/desktop/__init__.py,
```
       return _readfrom(_get_x11_vars() + "xprop -root _DT_SAVE_MODE", shell=1).strip().endswith(' = "xfce4"')
```
should perhaps be
````
       return _readfrom(_get_x11_vars() + "xprop -root _DT_SAVE_MODE", shell=1).decode(errors="ignore").strip().endswith(' = "xfce4"')
````
:)

**Comment by n1k0** [comment]
If that patch works for you, would you mind creating a PR?

**Comment by hashworks** [comment]
PR #61 fixes this for me and is a cleaner solution in my eyes.

**Comment by n1k0** [comment]
Fixed by #61.
